src/components/pages/AdminHome/AdminSidebar.js

```diff
@@ -45,15 +45,15 @@ function AdminSidebar() {
         </Menu.Item>
 
         <Menu.Item key="2" icon={<HomeOutlined />}>
-          <a href="/admin">Dashboard</a>
```

Comment:
can we add a `.map` here to make that coder super clean?

Reply:
That makes sense. As long as you have considered it already, I am good with the decided approach.